tests/resources/Vsphere-Util.robot

```diff
@@ -203,6 +203,7 @@ Add Host To VCenter
     \   ${out}=  Run  govc cluster.add -hostname=${host} -username=${user} -dc=${dc} -password=${pw} -noverify=true
     \   ${status}=  Run Keyword And Return Status  Should Contain  ${out}  OK
     \   Return From Keyword If  ${status}
+    \   Sleep  3 minutes
```

Comment:
In this case you are right, we are adding a 6.5 host to a 6.0 vc which is not allowed, I have already fixed that also in this #5468 but I would still like to not hammer a sick VC system with repeated calls without break.